### PR TITLE
moves next after numbers

### PR DESCRIPTION
**Pull request by dnoneill**
You can see the changes by changing https://github.com/projectblacklight/blacklight/blob/a8b5a3d3a2e8243c8a875d40528eae5c106d83bd/lib/blacklight/engine.rb#L71 to `blacklight9`

Screenshot 2024-08-14 at 11 34 18 AM

### CODE REVIEW

File: app/assets/stylesheets/blacklight/_pagination.scss [hunk 1]
```diff
@@ -14,3 +14,20 @@
...
+
+.link-pagination {
+  .disabled a {
+    pointer-events: none !important;
+    cursor: default;
+    color:gray;
+  }
+
+  .active span {
+    color: black;
```
Comment (jcoyne) [inline app/assets/stylesheets/blacklight/_pagination.scss:27]: can we set `--bs-pagination-active-color` instead?  Is there a reason we need to customize this here and not in the host app?  The host app may make different choices for which colors are most appropriate.

File: app/assets/stylesheets/blacklight/_pagination.scss [hunk 2]
```diff
@@ -14,3 +14,20 @@
     flex-wrap: wrap;
   }
 }
+
+
+.link-pagination {
+  .disabled a {
```
Comment (jcoyne) [inline app/assets/stylesheets/blacklight/_pagination.scss:20]: This rule should be unnecessary if you set `.page-item` on the node. https://getbootstrap.com/docs/5.3/components/pagination/#disabled-and-active-states

File: app/assets/stylesheets/blacklight/_pagination.scss [hunk 3]
```diff
@@ -14,3 +14,20 @@
...
+    pointer-events: none !important;
+    cursor: default;
+    color:gray;
+  }
+
+  .active span {
+    color: black;
+  }
+
+  .btn {
```
Comment (jcoyne) [inline app/assets/stylesheets/blacklight/_pagination.scss:30]: I think we should avoid this class in favor of `.page-link`

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/views/kaminari/blacklight9/_first_page.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<%# Link to the "First" page
+  - available local variables
+    url:           url to the first page
+    current_page:  a page object for the currently displayed page
+    num_pages:     total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+-%>
+<li class="page-item <%= 'disabled' if current_page.first? %>">
+  <%= link_to_unless current_page.first?, raw(t 'views.pagination.first'), url, :remote => remote, class: 'btn btn-link', aria: { label: t('views.pagination.aria.go_to_first_page') } %>
```
[the remark this answers is not in the document]
Comment (dnoneill) [reply app/views/kaminari/blacklight9/_first_page.html.erb:10]: The page-link makes it look like a button, which we don't want. I didn't realize the page-link was part of a bootstrap configuration so it looks like I should remove the page-item class.

File: app/views/kaminari/blacklight9/_prev_page.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  - available local variables
+    url:           url to the previous page
+    current_page:  a page object for the currently displayed page
+    num_pages:     total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+-%>
+<% if current_page.first? %>
+    <li class="page-item disabled">
+      <%= link_to raw(t 'views.pagination.previous'), '#', :rel => 'prev', :onclick=>'return false;', class: 'btn btn-link text-decoration-none', aria: { label: t('views.pagination.aria.go_to_previous_page') } %>
```
[the remark this answers is not in the document]
Comment (dnoneill) [reply app/views/kaminari/blacklight9/_prev_page.html.erb:11]: That is what is in the current pagination.

File: app/views/kaminari/blacklight9/_prev_page.html.erb [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    </li>
+<% else %>
+    <li class="page-item">
+      <%= link_to raw(t 'views.pagination.previous'), url, :rel => 'prev', :remote => remote, class: 'btn btn-link text-decoration-none', aria: { label: t('views.pagination.aria.go_to_previous_page') } %>
```
[the remark this answers is not in the document]
Comment (dnoneill) [reply app/views/kaminari/blacklight9/_prev_page.html.erb:15]: I don't know. This again was in the old code.

**Comment by jcoyne** [comment]
@dnoneill this requires a rebase.

**Pull request merged**